back/db/seeds/citizenlab.rb

```diff
@@ -85,6 +85,10 @@ def create_for_current_locales
       enabled: true,
       allowed: true
     },
+    survey_xact_surveys: {
```

Comment:
Not sure, but possibly, you'll also want to add this in create_tenant.rake? (to make it work for e2e tests and epic deployments)